shell/assets/translations/en-us.yaml

```diff
@@ -6131,6 +6131,7 @@ typeDescription:
   #       Should fit on one line.
   #       If you link to anything external, it MUST have
   #       target="_blank" rel="noopener noreferrer nofollow"
+  branding: "Branding allows for the users to change the overall branding of Rancher."
```

Comment:
I think we could probably say something more specific about the functionality of the branding page. Maybe along the lines of "Branding allows administrators to globally re-brand the UI by customizing the Rancher product name, logos and color scheme."

Reply:
Thanks for the feedback @mantis-toboggan-md ! 🙏